keyboards/tercy/rev004/readme.md

```diff
@@ -0,0 +1,35 @@
+# Tercy/rev004
+
+![Tercy/rev004](https://etlife.info/wp-content/uploads/2024/05/Tercy_rev004_keyboard.jpg)
```

Comment:
Please rehost these images on Imgur or similar image hosting sit.   Blogs like this are nvere a good long term solution.

Reply:
Thank you for your advice. I rehosted the images to Imgur.